go/vt/sqlparser/parse_test.go

```diff
@@ -3255,6 +3255,27 @@ func TestCreateTable(t *testing.T) {
 	}
 }
 
+func TestOne(t *testing.T) {
+	testOne := struct {
+		input, output string
+	}{
+		input:  "",
+		output: "",
+	}
+	if testOne.input == "" {
+		return
+	}
+	sql := strings.TrimSpace(testOne.input)
+	tree, err := Parse(sql)
```

Comment:
looks like the test is skipped with the if condition.

Reply:
Yes that is the intention. It is supposed to be used for testing one query while working and shouldn't have any other tests. Kinda like how we have testOne in the plan_tests